package.json

```diff
@@ -12,7 +12,7 @@
         "errors": "./node_modules/.bin/eslint --config .eslintrc --ext .jsx,.js \"src/js/**\" --quiet; exit 0;",
         "dev": "webpack --progress --config ./webpack/webpack.dev.config.js",
         "prod": "webpack --progress --config ./webpack/webpack.prod.config.js",
-        "travis": "npm audit --audit-level=moderate && ./node_modules/.bin/eslint --config .eslintrc --ext .jsx,.js \"src/js/**\" && ./node_modules/.bin/jest --runInBand;",
+        "travis": "npm audit --audit-level=moderate --production && ./node_modules/.bin/eslint --config .eslintrc --ext .jsx,.js \"src/js/**\" && ./node_modules/.bin/jest --runInBand;",
```

Comment:
Can you explain this update? I don't doubt you, just trying to learn. Will this allow us to use vulnerable packages that won't be caught until prod builds?